### DISCUSSION TITLE
Welcome to desktopPet Discussions!

### DISCUSSION THREAD

**Discussion by Adrianotiger** [Adrianotiger/desktopPet#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JesusGod-Pope666-Info** [comment]
Well, Darkijah here, remember the Sheep from my youth. And sorted it out for my website on a mobile. Use it to get more familiar with code overall.... Although I never seem to get that good with it, but I did get it working all 7 Sheep, that I then put on my website sorta hidden in my Sharing is Caring at the bottom of the screen on my website. Like why not.... And it is fun some times to just push it a couple of times and see them go around. Have one running around my test website as well, that I am working on off, to try to make something to replace my old design on my prime website.

I have some ideas to see if I can make the website a little more Sheep friendly, but we will see. Coding has never been my thing, but sometimes I get something done anyway with enough time.

I was thinking of the Sheep as a kind of Biblical Pass-over Lamb thing, although Aliens and abduction is kinda not that Christian but rather the opposite.... But.... Yea well.... I still have it on my webpage as a gimmick, if anyone wanna see how it is done and all, it possible to look into the source of my website and take whatever and use my settings for your own website. Took me some time to sort it out I recall, but by all means, if it can help get you 7 Sheep on your website quickly, then by all means take it all :) Sharing is Caring.

In the old days of the internet it was like a community, the expectation was just to take from each other without asking - went both ways of cause. Making something you expected others to just use it if they found it useful for something.

Anyway.... I don't think making animations and coding is for me on the Sheep, but Ideas tend to come up. So if someone wanna work on that, we could gather up a list of ideas for some more sheep animations.
